internal/daemon/generate.go

```diff
@@ -0,0 +1,9 @@
+//FIXME: This should not be necessary, but generating a generate-only
+// folder breaks the generation during build phase as per:
+// https://salsa.debian.org/go-team/packages/dh-golang/-/merge_requests/22
+
+//go:build generate
+
+//go:generate go generate -C testdata/grpctestservice
+
+package daemon
```

Comment:
I'll use this as a general comment, but it is valid for all the generate additions:

This is adding way too many `go:generate` lines and, while it can be tempting and powerful, I think is not a good practice to rely on this command too much. Maybe @didrocks can explain a bit better why.

Reply:
I think this one is a leftover anyway as we don’t want to run generate (see above) during package build.